JumpscaleCore/data/schema/SchemaFactory.py

```diff
@@ -73,6 +73,10 @@ def schema_cache_remove(self, url):
             self.schemas_loaded.pop(url)
             if s._md5 in self.schemas_md5:
                 self.schemas_md5.pop(s._md5)
+            if self.meta._data["url"].get(url):
+                self.meta._data["url"].pop(url)
```

Comment:
if this is a regular dict, and you doubt that the key exists while popping you can just do .pop(url, None) to avoid getting a keyerror